application/models/__init__.py

```diff
@@ -172,6 +173,54 @@ def get_exchanges_game_29():
         return data
 
 
+def get_exchanges_game_31():
```

Comment:
This function and "get_heart_to_heart_exchanges()" repeat a chunk of code that could be moved to its own function.